widget/src/pick_list.rs

```diff
@@ -570,7 +560,11 @@ where
             &mut state.menu,
             options,
             &mut state.hovered_option,
-            &mut state.last_selection,
+            |option| {
+                state.is_open = false;
+
+                (on_selected)(option)
+            },
```

Comment:
I was quite surprised when this compiled. Rust seems to be capable to partially capture fields of a struct in a closure now.

Reply:
It makes a big difference!